rake_nltk/rake.py

```diff
@@ -184,18 +190,26 @@ def _generate_phrases(self, sentences):
     def _get_phrase_list_from_words(self, word_list):
         """Method to create contender phrases from the list of words that form
         a sentence by dropping stopwords and punctuations and grouping the left
-        words into phrases. Ex:
+        words into phrases. All the phrases should have a length in the range
```

Comment:
Maybe change the wording to `Only phrases in the given range would be considered to build co-occurrence matrix.